Configure.pl

```diff
@@ -95,7 +95,7 @@ sub uniq {
 $args{debug}        = 3 if defined $args{debug} and $args{debug} eq "";
 
 # Relocatability is not supported on AIX and OpenBSD.
-$args{'no-relocatable'} = 1 if $^O eq 'aix' || $^O eq 'openbsd';
+$args{'relocatable'} = 0 if $^O eq 'aix' || $^O eq 'openbsd';
```

Comment:
If somebody passed it explicitly, and we can't do it, should we not complain rather than silently override their choice?

Reply:
Changed accordingly. Thanks for the hint!